lib/rester/client.rb

```diff
@@ -138,5 +158,17 @@ def _parse_json(data)
         {}
       end
     end
+
+    def _log_req_res(type, verb, path, status=nil)
+      arrow_str = type == :request ? '->' : '<-'
+      log_str = "[#{Rester.service_name}] #{arrow_str} #{_producer_name} - " \
+        "#{verb.upcase} #{path}"
+      log_str << " #{status}" if status
+      _log_with_correlation_id(:info, log_str)
+    end
```

Comment:
I would change `status` to `msg` here, so we can pass in a generic message, instead of only the status.  Maybe the message could be `Status=200` or something.

And then, we'll want to use that `msg` to specified `time out` or `circuit open`.